src/resources/Sources/SourcesSubInterfaces/Information.ts

```diff
@@ -32,12 +32,19 @@ export interface NextOperation {
 
 export interface SourceStatus {
     allowedOperations?: ActivityOperation[];
+    /**
+     * The new type status (includes statuses of transition).
+     */
+    extendedCurrentStatus?: SourceStatusType;
```

Comment:
I wonder if the actual type (`SourceStatusType`) should be different to represent the reality. It would make the difference cleaner than just the jsdocs

Reply:
I check with APIste ;)